lib/bittrex/deposit.rb

```diff
@@ -9,7 +9,7 @@ def initialize(attrs = {})
       @quantity = attrs['Amount']
       @currency = attrs['Currency']
       @confirmations = attrs['Confirmations']
-      @executed_at = Time.parse(attrs['LastUpdated'])
+      @executed_at = Bittrex::extract_timestamp(attrs['LastUpdated'])
```

Comment:
Please invoke this off the class directly: `Bittrex.extract_timestamp`. Or preferably, move `extract_timestamp` into a helper module so the method isn't on the parent namespace eg `Bittrex::Helpers.extract_timestamp`

Reply:
Extracted to helper module.